test/unit/main.c

```diff
@@ -66,25 +66,14 @@ int main(int argc, char **argv) {
     qsort(base, argc - 1, sizeof(char *), compare_args);
   }
 
-  int tsize = test_suite_size;
-
-#ifdef TEST_DISABLE_TCP
-  JNXLOG(LDEBUG,"Running with disabled TCP tests");
-  tsize--;
-#endif
-#ifdef TEST_DISABLE_IPC
-  JNXLOG(LDEBUG,"Running with disabled IPC tests");
-  tsize--;
-#endif
-#ifdef TEST_DISABLE_UDP
-  JNXLOG(LDEBUG,"Running with disabled UDP tests");
-  tsize--;
+#ifdef TEST_DISABLE_IPV6
+  JNXLOG(LDEBUG,"Running with disabled IPv6 tests");
```

Comment:
I am suprised this would work if you are using `test_suite_size` without accounting for the removed test